packages/block-library/src/gallery/edit.js

```diff
@@ -250,7 +250,7 @@ class GalleryEdit extends Component {
 				imageSizes,
 				( { slug } ) => some( resizedImages, ( sizes ) => ( sizes[ slug ] ) )
 			),
-			( { name, slug } ) => ( { value: slug, label: name } )
+			( { name, slug, label } ) => ( { value: slug, label: label || name } )
```

Comment:
Hi @mkevins, any specific reason for on mobile the property part of imageSizes is named label instead of name?

Reply:
Hi @jorgefilipecosta :wave: Thanks for circling back on this one. I believe we are just using [the defaults](https://github.com/WordPress/gutenberg/blob/24c207a11f8797d7da1cec26ea290545904bd406/packages/block-editor/src/store/defaults.js#L131) here.

Reply:
I tried to understand the issue, it seems we had a naming problem. Core passes image sizes with the name of the image size named "name", while on Gutenberg defaults we named it "label". I think the way to go would be to rename the defaults in https://github.com/WordPress/gutenberg//blob/24c207a11f8797d7da1cec26ea290545904bd406/packages/block-editor/src/store/defaults.js#L132 to "name" and update the existing code in mobile to rely on "name" as the web does.
What do you think?

Reply:
Thanks for looking into that @jorgefilipecosta :+1: I've updated the defaults to use `name` instead of `label`. Ready for another pass.